### PR TITLE
Added utility methods for masking data in logs

### PR DESCRIPTION
**Pull request by mattiasavelin**
Added utility methods for masking sensitive data in logs

### CODE REVIEW

File: pkg/logging/utils.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package logging
+
+import (
+	"regexp"
+	"strings"
+)
+
+// MaskSSN finds and masks any SSN in the format YYYYMMDDXXXX in the input string
+func MaskSSN(input string) string {
```
Comment (alexpalyan) [inline pkg/logging/utils.go:9]: dentech-floss is placement for open source libs
I suggest clarifying that this is specifically about the Swedish SSN.

Comment (mattiasavelin) [reply pkg/logging/utils.go:9]: That is a very good point! 👍 Will fix

Comment (mattiasavelin) [reply pkg/logging/utils.go:9]: But then again the maskPhone currently primarily handles Swedish phone nbrs too. Maybe this does not belong in the floss-logging. 🤔 ...will reconsider this.

Comment (mattiasavelin) [reply pkg/logging/utils.go:9]: After marinating on this over the weekend I think maybeit's better for now at least to keep this in the service instead. Then maybe later we can refine it be general enough to belong here.

Will close this PR.

**Pull request closed without merging**